core-web/libs/dotcms-scss/angular/dotcms-theme/components/_splitbutton.scss

```diff
@@ -1,43 +1,294 @@
 @use "variables" as *;
 
-.p-splitbutton > .p-button:disabled {
-    border-color: rgba(0, 0, 0, 0.01);
-}
+// Split Button Sizes
+.p-splitbutton {
+    .p-splitbutton-menubutton {
+        border-radius: 0 0.375rem 0.375rem 0;
+        height: 2.5rem;
+        padding: 0 $spacing-1;
+        border: none;
 
-.p-splitbutton.p-button-secondary > .p-splitbutton-menubutton:not(:disabled) {
-    border-right: 1px solid $color-palette-secondary-500;
-}
+        .p-button-icon,
+        .pi {
+            color: $white;
+            font-size: $font-size-lg;
+            line-height: 140%;
+        }
+    }
 
-.p-splitbutton.p-button-secondary > .p-splitbutton-menubutton:not(:disabled):hover {
-    border-right: 1px solid $color-palette-primary;
-}
+    .p-splitbutton-defaultbutton {
+        gap: $spacing-1;
+        padding: 0 $spacing-3;
+        height: 2.5rem;
+        border-radius: 0.375rem 0 0 0.375rem;
```

Comment:
Border radius variable for `0.375rem`?

Reply:
Will work on it, marked as resolved since I have a similar comment about this